Lib/email/message.py

```diff
@@ -134,6 +134,10 @@ def __str__(self):
         """
         return self.as_string()
 
+    def __repr__(self):
+        return f"{self.__class__.__name__} with {len(self._headers)} " \
+               f"headers and Content-Type {self._default_type}"
```

Comment:
This doesn't match the suggestion in the issue.  self._default_type is also wrong, since that's just the default, not the actual content-type of the message (which comes from the headers).

What exactly goes in the repr should be discussed further in the issue.